src/components/gallery/HdGalleryCarousel.vue

```diff
@@ -294,7 +294,6 @@ export default {
       z-index: 1;
 
     img {
-      width: 100%;
       height: 100%;
       object-fit: cover;
     }
```

Comment:
Hey mate! Nice approach! Removing the `width` property works, but what do you think using `object-fit: contain` instead of `object-fit: cover` to fix the issue? https://www.loom.com/i/575ff9863d0546e8ac08518997619f59 - Because `object-fit: cover` is the main reason the image looks covered within the element.

Nevertheless, I'm okay with your approach as well if removing the `width` makes more sense to you! But, shouldn't we remove the `object-fit` property completely as it becomes unnecessary once we remove the `width` property? Here is the same result without having `width` & `object-fit`. https://www.loom.com/i/7f2d23416c9d462db6a933247d42ae8b